index.html

```diff
@@ -61,7 +61,7 @@ <h4>Stable</h4>
     <div class="row">
       <div class="col-sm-12">
         <a id="asf-current-event-logo" href="https://www.apache.org/events/current-event.html"><img alt="ASF Current Events" class="img-responsive center-block" src="https://www.apache.org/events/current-event-234x60.png"/></a><br/>
-        <a id="accumulo-summit-logo" href="http://accumulosummit.com/"><img alt="Accumulo Summit" class="img-responsive center-block" src="{{ site.baseurl }}/images/accumulo-summit-2017-logo-horizontal.png"/></a>
```

Comment:
Should `accumulo-summit-2017-logo-horizontal.png` be also deleted?

Reply:
It is used in the blog post for last years summit.  So it should stay around unless we delete that blog post.